rdt/transformers/categorical.py

```diff
@@ -185,16 +181,6 @@ def _transform(self, data):
 
         return self._transform_by_row(data)
 
-    def _normalize(self, data):
```

Comment:
@npatki I remember the doc said something about allowing for normalization as an option, but the issue doesn't mention it. Are we just always going to clip now?

Reply:
Yes always clip. If the model is generating too many out-of-bounds values, that is an issue with the model.

Normalizing can have other unintended consequences.